### PR TITLE
[master] Fix heading level of "creates"

### PR DESCRIPTION
**Pull request by OrangeDog**
### What does this PR do?
Changes the "creates" heading to match all the other headings under "Altering States".

### CODE REVIEW

**Review by s0undt3ch — APPROVED** [review]

File: doc/ref/states/requisites.rst [hunk 1]
```diff
@@ -1053,8 +1053,8 @@ if the gluster commands return a 0 ret value.
 
 .. _creates-requisite:
 
-Creates
--------
+creates
+~~~~~
```
Comment (s0undt3ch) [inline doc/ref/states/requisites.rst:1057]:
```suggestion
creates
~~~~~~~
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: doc/ref/states/requisites.rst (hunk 1 above)
Comment (s0undt3ch) [reply doc/ref/states/requisites.rst:1057]:
```
/home/runner/work/salt/salt/doc/ref/states/requisites.rst:1057: WARNING: Title underline too short.
```

**Review by Ch3LL — APPROVED** [review]

**Comment by s0undt3ch** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|3006.x|[](https://github.com/saltstack/salt/pull/65277)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by s0undt3ch** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|3006.x|[](https://github.com/saltstack/salt/pull/65277)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**